components/org.wso2.carbon.identity.api.server.flow.management/org.wso2.carbon.identity.api.server.flow.management.v1/src/main/resources/flow.yaml

```diff
@@ -147,6 +193,49 @@ components:
           items:
             $ref: '#/components/schemas/Step'
 
+    FlowMetaResponse:
+      type: object
+      description: Metadata for a given flow type
+      properties:
+        flowType:
+          type: string
+          example: PASSWORD_RECOVERY
+        supportedExecutors:
+          type: array
+          items:
+            type: string
+          example:
+            - UserResolveExecutor
+            - PasswordProvisioningExecutor
+            - EmailOTPExecutor
+        connectorConfig:
```

Comment:
Shall we separate this response into a `GeneralResponse` model, which contains all common properties required for all flow types, and move the flow-specific configurations into separate response models for each specific flow type? For example, the connections property is currently relevant only to the registration flow, so it would be better to move its metadata into a separate model (ConnectionMetaModel). Then, we can create a dedicated `RegistrationFlowResponse` model that references both the `GeneralResponse` and the `ConnectionMeta`, while other flow types can reference only the `GeneralResponse`.